src/adagrams.js

```diff
@@ -1,15 +1,214 @@
+const LETTERPOOL = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
+
+const createLetterList = () => {
+  const letters = [];
+  const letterList = [];
+  for (const [key, value] of Object.entries(LETTERPOOL)) {
+    letters.push(key.repeat(value)) ;
+  }
+  for (const ele of letters) {
+    for (const letter of ele) {
+      letterList.push(letter);
+    }
+  }
```

Comment:
This line could be mitigated with using the Javascript spread syntax like so `letters.push(...key.repeat(value))`